templates/company-board/package.json

```diff
@@ -25,9 +25,9 @@
     "docker:wait-gas": "truffle exec ./node_modules/@aragon/templates-shared/scripts/sleep-until-gaslimit.js --network devnet 6900000",
     "deploy:rpc": "npm run compile && truffle exec ./scripts/deploy.js --network rpc",
     "deploy:devnet": "npm run compile && truffle exec ./scripts/deploy.js --network devnet",
-    "deploy:staging": "npm run compile && truffle exec ./scripts/deploy.js --network rinkeby --ens 0xfe03625ea880a8cba336f9b5ad6e15b0a3b5a939",
-    "deploy:rinkeby": "npm run compile && truffle exec ./scripts/deploy.js --network rinkeby --ens 0x98Df287B6C145399Aaa709692c8D308357bC085D",
-    "deploy:ropsten": "npm run compile && truffle exec ./scripts/deploy.js --network ropsten --ens 0x6afe2cacee211ea9179992f89dc61ff25c61e923",
+    "deploy:staging": "npm run compile && truffle exec ./scripts/deploy.js --network rinkeby --ens 0xfe03625ea880a8cba336f9b5ad6e15b0a3b5a939 --dao-factory 0x137676c1941c8b12d18b12eb2215d1361aee20e0 --mini-me-factory 0xc31d4145ca14227b2d780dd51cea0923dae609da",
+    "deploy:rinkeby": "npm run compile && truffle exec ./scripts/deploy.js --network rinkeby --ens 0x98Df287B6C145399Aaa709692c8D308357bC085D --dao-factory 0x137676c1941c8b12d18b12eb2215d1361aee20e0 --mini-me-factory 0xc31d4145ca14227b2d780dd51cea0923dae609da",
+    "deploy:ropsten": "npm run compile && truffle exec ./scripts/deploy.js --network ropsten --ens 0x6afe2cacee211ea9179992f89dc61ff25c61e923 --dao-factory 0x56aee4a6e24e36fa6dc653d81dfdf76aa149f774 --mini-me-factory 0xc841c1b35bdc8f7c35b7703cd797525143069fbb",
```

Comment:
For Rinkeby and Ropsten, can we use the same DAOFactory and MiniMeTokenFactory addresses as 0.7?

- Rinkeby:
  - DAOFactory: `0xfdef49fbfe37704af55636bdd4b6bc8cd19143f6`
  - MiniMeTokenFactory: `0x6ffeb4038f7f077c4d20eaf1706980caec31e2bf`
- Ropsten:
  - DAOFactory: `0x3f2aa9dd22e97070518ba7988fe9b8724129d497`
  - MiniMeTokenFactory: `0x1ce5621d386b2801f5600f1dbe29522805b8ac11`

Reply:
- Staging:
  - DAOFactory: `0xbca2c99aa7018edcde60e6c5744f68003112535d `
  - MiniMeTokenFactory: `0xac443f983c38c02149ce2ffcff8b7de90dccf77c`
- Mainnet:
  - DAOFactory: `0xc29f0599df12eb4cbe1a34354c4bac6d944071d1`
  - MiniMeTokenFactory: `0x909d05f384d0663ed4be59863815ab43b4f347ec`